t/api/02-assets.t

```diff
@@ -98,8 +96,8 @@ delete $t->tx->res->json->{$_} for qw/t_updated t_created/;
 $t->json_is('' => $listing->[0], "asset correctly entered by id");
 $t->get_ok('/api/v1/assets/iso/' . $iso2)->status_is(404, 'iso does not exist');
 
-$t->post_ok('/api/v1/assets', form => {type => 'iso', name => $iso2})->status_is(200, 'second asset posted')
-  ->json_is('/id', $listing->[1]->{id}, "asset has corect ID");
+$t->post_ok('/api/v1/assets', form => {type => 'iso', name => $iso2})->status_is(200, 'second asset posted');
+$listing->[1]->{id} = $t->tx->res->json->{id};
```

Comment:
Maybe you could check whether `$listing->[0]` and `$listing->[1]` are different IDs.

Reply:
Nice idea! Added a check!